content/docs/Konveyor/Kai/IDESettings/proc_configuring-developer-lightspeed-ide-settings.md

```
@@ -0,0 +1,138 @@
+---
+title: "Configuring the LLM provider settings"
+date: 2025-12-05T14:58:52-06:00
+draft: false
+---
+After you install the Konveyor extension in Visual Studio (VS) Code, you must provide your large language model (LLM) credentials to activate Konveyor AI settings in Visual Studio (VS) Code.
+
+Konveyor AI settings are applied to all AI-assisted analysis that you perform by using the Konveyor extension. The extension settings can be broadly categorized into debugging and logging, Konveyor AI settings, analysis related settings, and Solution Server settings.
+
+**Prerequisites**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)